stories/index.stories.js

```diff
@@ -364,7 +364,7 @@ storiesOf("Carousel", module)
     return (
       <Carousel
         infinite
-        partialVisbile="right"
+        partialVisible="right" // should be boolean ideally
```

Comment:
Yeah, this should be changed to boolean

Reply:
@YIZHUANG How do you do the partial visibility to the left now?

Reply:
@ardian-c I think partialVisibility on the left can be achieved easily via centerMode (with infinite).. i dont think partial visibility on the left without centerMode(with infinite) makes sense; but if you absolutely need it you could try passing a negative additionalTransform ..

Reply:
@abhinavdalal-iconnect thank you, will try it with negative `additionalTransform`, as I'm not sure if I understand correctly `centerMode` with infinite.